Classes/CGROSS/Drinkaccounting/Controller/PurchasePositionController.php

```diff
@@ -23,6 +23,12 @@ class PurchasePositionController extends DefaultController {
 	protected $purchaseRepository;
 
 	/**
+	 * @Flow\Inject
+	 * @var \TYPO3\Flow\Persistence\PersistenceManagerInterface
+	 */
+	protected $persistenceManager;
+
```

Comment:
Please remove, already inject by DefaultController from which this controller inherits